ui/apps/platform/src/Containers/NetworkGraph/simulation/DeploymentScopeModal.tsx

```diff
@@ -38,6 +36,13 @@ function DeploymentScopeModal({ deployments, isOpen, onClose }: DeploymentScopeM
                 </Button>,
             ]}
         >
+            {entityScope.hasAppliedDeploymentFilters && (
+                <Alert
+                    isInline
+                    variant="info"
+                    title="The deployment scope for generated policies may be further reduced by filters applied on this page"
```

Comment:
Dave, can you double check if axe DevTools reports an issue about heading levels?

You might have noticed in a recent pull request that when `title` attribute is really the content, `component="div"` is more accessible and semantic. Even if we corrected the level, it does not make sense for screen reader to announce the sentence as if it is a heading for navigation.

Reply:
Positive feedback: in this case, passive voice seems like a helpful choice so sentence starts with key phrases **The deployment scope** and **for generated policies**

Some rewording might make this sentence even clearer.

Somehow **filters applied on this page** stopped me.

The **second** passive phrase is one reason, and here are 2 additional observations:
1. For **on this page** is this the only landing page that does not have a visible `h1` heading?
    Even without a page heading, navigation sidebar has the title.
    Therefore, what do you think about **Network Graph** instead of **this page**?
2. Generic search filter has only a **Filter deployments** placeholder, but not a label (as usual).
    What are pro and con of rewriting with implicit label: **by Network Graph deployment filter**
    That first draft might need additional iterations, because it is ambiguous with **Deployments** of the triple filter. Which does not cause this problem, correct?